doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -313,15 +317,61 @@ The Platform is assumed to provide configurable alerting and notification capabi
+
+#### 2.2.6.9. IaaC security (source [RM7.9.9](../../../ref_model/chapters/chapter07.md#799-iaac---secure-design-and-architecture-stage-requirements))
+
+**Secure Code Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.code.001 | SAST -Static Application Security Testing **must** be applied during Secure Coding stage triggered by Pull, Clone or Comment trigger. | Security testing that analyses application source code for software vulnerabilities and gaps against best practices. Example: open source OWASP range of tools.|
+
+<p align="center"><b>Table 2-14:</b> Reference Model Requirements: IaaC Security Requirements, Secure Code Stage </p>
+
+**Continuous Build, Integration and Testing Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.bld.003 | Container and Image Scan **must** be applied during the Continuous Build, Integration and Testing stage triggered by Package trigger. | Example: A push of a container image to a container registry may trigger a vulnerability scan before the image becomes available in the registry.  |
+
+<p align="center"><b>Table 2-15:</b> Reference Model Requirements: IaaC Security Requirements, Continuous Build, Integration and Testing Stage </p>
+
+**Continuous Delivery and Deployment Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.del.001 | Image Scan **must** be applied during the Continuous Delivery and Deployment stage triggered by Publish to Artifact and Image Repository trigger. | Example: GitLab uses the open source Clair engine for container scanning.|
+| sec.del.002 | Code Signing **must** be applied during the Continuous Delivery and Deployment stage triggered by Publish to Artifact and Image Repository trigger. | Code Signing provides authentication to assure that downloaded files are form the publisher named on the certificate.  |
+| sec.del.004 | Component Vulnerability Scan **must** be applied during the Continuous Delivery and Deployment stage triggered by Instantiate Infrastructure trigger. | The vulnerability scanning system is deployed on the cloud platform to detect security vulnerabilities of specified components through scanning and to provide timely security protection. Example: OWASP Zed Attack Proxy (ZAP). |
+
+<p align="center"><b>Table 2-16:</b> Reference Model Requirements: IaaC Security Requirements, Continuous Delivery and Deployment Stage </p>
 
+**Runtime Defence and Monitoring Requirements**
 
-<p align="center"><b>Table 2-6:</b> Reference Model Requirements: Cloud Infrastructure Security Requirements</p>
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.run.001 | Component Vulnerability Monitoring **must** be continuously applied during the Runtime Defence and Monitoring stage. | Security technology that monitors components like virtual servers and assesses data, applications, and infrastructure for security risks.|
```

Comment:
@karinesevilla Should this also have a requirement to fix identified vulnerabilities?

Reply:
@pgoyal01 Yes I think so, but as sais before, it must be discussed in RM first. Could it be kept for Lakelse?

Reply:
@karinesevilla Agree. Needs an RM issue for Lakelse release. This can then be resolved. Thanks

Reply:
@goyal Issue#2481created